frontend/src/app/modules/boards/boards-root/boards-root.component.ts

```diff
@@ -7,6 +7,7 @@ import {QueryUpdatedService} from "core-app/modules/boards/board/query-updated/q
 import {BoardAssigneeActionService} from "core-app/modules/boards/board/board-actions/assignee/assignee-action.service";
 import {BoardSubprojectActionService} from "core-app/modules/boards/board/board-actions/subproject/subproject-action.service";
 import {BoardSubtasksActionService} from "core-app/modules/boards/board/board-actions/subtasks/board-subtasks-action.service";
+import {BoardPriorityActionService} from "core-app/modules/boards/board/board-actions/priority/priority-action.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/c84278b48bf04ab4ece2ead5a2251769516e329b/frontend/src/app/modules/boards/boards-root/boards-root.component.ts#L11"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>